elyra/contents/handlers.py

```diff
@@ -38,28 +38,22 @@ async def get(self, path=''):
         path = path or ''
         path = url_unescape(path)
 
-        type = self.get_query_argument('type', default=None)
-        if type not in {None, 'file', 'notebook'}:
-            raise web.HTTPError(400, u'Type %r is invalid' % type)
-
         self.log.debug("Parsing file: %s", path)
 
         try:
             model = await self.operation_parser(path)
         except FileNotFoundError as fnfe:
             model = dict(title=str(fnfe))
             self.set_status(404)
```

Comment:
Since this leads to a `raise` there's no need to set `model` or the status.  (Same goes for IADE.)

Reply:
I swore I tested this but I just tried it again and I was clearly must not have tested the right version! Sorry about that :)

Reply:
No worries.